ui/src/main/java/com/epam/eco/kafkamanager/ui/permissions/PermissionController.java

```diff
@@ -47,20 +57,28 @@
 
     private static final int PAGE_SIZE = 10;
 
+    public static final String PERMISSION_CREATE_VIEW = "permission_create";
+
     public static final String VIEW = "permissions";
     public static final String ATTR_PAGE = "page";
     public static final String ATTR_SEARCH_QUERY = "searchQuery";
     public static final String ATTR_TOTAL_COUNT = "totalCount";
+    public static final String ATTR_DEFAULT_RESOURCE_TYPE = "defaultResourceType";
+    public static final String ATTR_DEFAULT_OPERATION = "defaultOperation";
+    public static final String ATTR_DEFAULT_PERMISSION_TYPE = "defaultPermissionType";
 
     public static final String MAPPING = "/permissions";
+    public static final String MAPPING_PERMISSION = MAPPING + "/{resourceType}/{resourceName}/{permissionType}/{operation}/{principal}/{host}";
     public static final String MAPPING_EXPORT = MAPPING + "/export";
+    public static final String MAPPING_CREATE = "/permission_create";
+    public static final String MAPPING_DELETE = MAPPING_PERMISSION + "/delete";
 
     @Autowired
     private KafkaManager kafkaManager;
 
-    @RequestMapping(value=MAPPING, method=RequestMethod.GET)
+    @RequestMapping(value = MAPPING, method = RequestMethod.GET)
```

Comment:
Please, do not apply this kind of changes, because it is just additional confusion.